src/sdc11073/wsdiscovery/__init__.py

```diff
@@ -0,0 +1,14 @@
+# noqa: D104
+from .common import MULTICAST_PORT
+from .wsdimpl import (
+    MATCH_BY_LDAP,
+    MATCH_BY_STRCMP,
+    MATCH_BY_URI,
+    MATCH_BY_UUID,
+    WSDiscovery,
+    WSDiscoverySingleAdapter,
+    match_scope,

```

Comment:
please also export `Service` class and `filter_services` and `matches_filter` methods

Reply:
why?